### DISCUSSION TITLE
How do I get people to sponsor me?

### DISCUSSION THREAD

**Discussion by intergalacticcoding** [community/community#108209] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

title.

**Comment by Alchemistpotion** [comment]
Make code that people want to put money in!

**Comment by ettaboyle** [comment]
@intergalacticcoding & @Alchemistwithpotion - This posts is similar to a number of posts we have seen related to achievement or badge hunting and adds no value to the community. This behavior violates the Community Code of Conduct and both of your accounts are being suspended for 7 days as a warning. If you again violate the code of conduct you will be blocked from this organization indefinitely. 

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

----

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).
